src/cogs/zigzag.py

```diff
@@ -65,19 +90,31 @@ async def oldest(self, ctx: Context):
                 selectView = PostOptionsView(thread, canidate_ids, candidate_image)
                 await ctx.send(message, view=selectView)
 
+            # Check if we are at our max number of threads
+            if num_found_threads >= MAX_NUM_FEEDBACK_THREADS:
+                self.most_recent_date = thread.archive_timestamp
+                break
+        
+        if num_found_threads == 0:
+            if not archived_thread_found:
+                await ctx.send("Found no feedback threads active before {}.\nTry re-running with `reset` or a specific date `MM/DD/YY`".format(self.most_recent_date))
+                return
+            await ctx.send("Found no feedback threads between {} and {}.\nRun again to search later, or you can run again with `reset` or a specific date `MM/DD/YY`".format(self.most_recent_date,
+                                                                                                                                          thread.archive_timestamp))
```

Comment:
For sanity's sake, maybe limit lines to 100 chars

Reply:
Good call, fixed